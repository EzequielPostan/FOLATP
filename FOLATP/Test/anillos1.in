loadKB( Test/anillos.kb )
conclusion( Odia(Frodo,Sauron) )
runTo( Test/anillos1.out )
exit
