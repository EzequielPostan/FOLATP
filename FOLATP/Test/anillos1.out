{}
│
├─ {~Hobbit(Frodo)}
│ │
│ ├─ {~Hobbit(Frodo), ~IntentaDestruir(Frodo, Sauron)}
│ │ │
│ │ ├─ {EsLeal(Frodo, Sauron), ~Hobbit(Frodo)}
│ │ │ │
│ │ │ ├─ {EsLeal(Frodo, Sauron), ~ViveEn(Frodo, Comarca)}
│ │ │ │ │
│ │ │ │ ├─ {EsLeal(Frodo, Sauron), ~ViveEn(Frodo, TierraMedia)}
│ │ │ │ │ │
│ │ │ │ │ ├─ {~Odia(Frodo, Sauron)}
│ │ │ │ │ │
│ │ │ │ │ └─ {EsLeal(x1, Sauron), Odia(x1, Sauron), ~ViveEn(x1, TierraMedia)}
│ │ │ │ │
│ │ │ │ └─ {ViveEn(x1, TierraMedia), ~ViveEn(x1, Comarca)}
│ │ │ │
│ │ │ └─ {ViveEn(x1, Comarca), ~Hobbit(x1)}
│ │ │
│ │ └─ {~EsLeal(x1, x2), ~IntentaDestruir(x1, x2)}
│ │
│ └─ {IntentaDestruir(Frodo, Sauron)}
│
└─ {Hobbit(Frodo)}
